park-dashboard/src/Components/Campgrounds/CampgroundsChart.jsx

```diff
@@ -0,0 +1,72 @@
+import React from "react";
+import { Campgrounds } from "./Campgrounds";
+import { Chart as ChartJS, ArcElement, Tooltip, Legend } from "chart.js";
+import { Pie } from "react-chartjs-2";
+
+const CampgroundsChart = () => {
+  const campgroundsData = Campgrounds();
+
+  const stateWiseCampgrounds = campgroundsData
+    .map((camp) => {
+      const physicalAddress = camp.addresses.find(
+        (address) => address.type === "Physical"
+      );
+
+      return {
+        address: physicalAddress ? physicalAddress.stateCode : "",
+      };
+    })
+    .filter((camp) => camp.address !== "");
+
+  const counts = stateWiseCampgrounds.reduce((acc, camp) => {
+    const stateCode = camp.address;
+    acc[stateCode] = (acc[stateCode] || 0) + 1;
+    return acc;
+  }, {});
+
+  const filteredCounts = Object.entries(counts).filter(
+    ([state, count]) => count >= 10
+  );
+
+  const stateCodes = filteredCounts.map(([state, count]) => state);
```

Comment:
Did you try using Object.keys and Object.values here instead of map function?

Reply:
Hey Juhi, filteredCounts is an array that is why I am using map here and not able to use Object.keys and values.